README.md

```diff
@@ -13,7 +13,7 @@ F# support in Rider makes use of open source software, most notably [FSharp.Comp
 
 ### Requirements
 
-* [.NET Framework 4.6.1 Developer Pack](https://www.microsoft.com/en-us/download/details.aspx?id=40772)
+* [.NET Framework 4.6.1 Developer Pack](https://www.microsoft.com/net/download/thank-you/net461-developer-pack)
```

Comment:
I think it would be better to have a download page where you start a download explicitly so it doesn't start automatically.